target/linux/ath79/dts/qca9533_mikrotik_routerboard-lhg-hb.dtsi

```diff
@@ -67,12 +67,23 @@
 	};
 };
 
-&eth1 {
+&eth0 {
+	status = "okay";
+
+	phy-handle = <&swphy0>;
+
 	gmac-config {
 		device = <&gmac>;
+		switch-phy-swap = <1>;
 	};
 };
 
+&eth1 {
+	status = "okay";
```

Comment:
eth1 is already enabled in qca953x, so you only need the compatible for this one here.

Reply:
Line removed, thanks.